pandas/_libs/tslibs/period.pyx

```diff
@@ -1408,6 +1408,30 @@ cdef class _Period(object):
 
     @property
     def qyear(self):
+        """
+        Get the year component of the Period.
+
+        Returns
+        -------
+        int --> qyear : int
+
+        See Also
+        --------
+        Period.year : Return the year of the date
+        Period.pqyear : Return the year of the period
+
+        Examples
+        --------
+        >>> p = pd.Period('2014Q1', freq="Q")
+        >>> p.qyear
+        2014
+
```

Comment:
It'd be good to add a bit of text explaining the difference here. Something like

    The fiscal-quarter year can differ from the calendar year.